python/main_loop.py

```diff
@@ -216,6 +217,22 @@ def createNewLocalPath(sailbot, maxAllowableRuntimeSeconds, desiredHeadingPublis
         # Publish local path
         localPathPublisher.publish(msg.path(localPath.getLatlons()))
 
+        # Format and log local path
+        latLonsToLog = "Path latlons are:"
+
+        for i in range(len(localPath.getLatlons())):
+            if(i % LOGGED_LATLONS_PER_LINE == 0):
+                latLonsToLog += "\n"
+
+            latLon = localPath.getLatlons()[i]
+            latLonsToLog += "(lat: {}, lon: {})".format(latLon.lat, latLon.lon)
```

Comment:
Looks very clear and easy to understand
![image](https://user-images.githubusercontent.com/26510814/137227550-bc5d109d-a456-4396-b0c6-0f275aaad101.png)